CMakeLists.txt

```diff
@@ -251,8 +251,8 @@ ${LOGGING_H_FILE}
 ./inc/azure_c_shared_utility/doublylinkedlist.h
 ./inc/azure_c_shared_utility/gballoc.h
 ./inc/azure_c_shared_utility/gbnetwork.h
-./inc/azure_c_shared_utility/gb_stdio.h
-./inc/azure_c_shared_utility/gb_time.h
+# ./inc/azure_c_shared_utility/gb_stdio.h
```

Comment:
rather than just commenting these out, could you make them conditional compiles?
So that if someone did start using GB_STDIO_INTERCEPT or GB_TIME_INTERCEPT the the needed headers would still be added?

Reply:
Rolled back my change and cherry picked the commit from the base repo that fixes this.